capstone-frontend/src/components/SearchBookModal.jsx

```diff
@@ -0,0 +1,238 @@
+                        <h5 className="mt-4"> {book.title} </h5>
+                        <p className="my-1"> {book.authors.join(', ')} </p>
+                        {this.state.addedBooksIDs.includes(book.id) ?
+                          <Button className="my-5" variant="danger" onClick={() => this.removeBookFromList(book)}>Remove Book</Button>
+                          :
+                          <Button className="my-5" onClick={() => this.addBookToList(book)}>Add to Booklist</Button>}
+                      </Col>
+                    )}
+                  </Row>
+                </div>
+              }
+
+              {
+                (this.state.addedBooks.length !== 0) && (
+
+                  (this.props.type === "club")
+                    ?
+                    <div>
+                      <h2 className="text-center my-4 px-4 ">Added Book</h2>
+                      <Row className="text-center px-3">
+                        {
+                          <Col className="px-2 my-0 border" key={this.state.addedBooks[0].id}>
+                            <img className="img-responsive mt-4 p-0 rounded" src={this.state.addedBooks[0].thumbnailLink} alt={this.state.addedBooks[0].title} />
+                            <h5 className="my-4"> {this.state.addedBooks[0].title} </h5>
+                            <p className="my-1"> {this.state.addedBooks[0].authors.join(', ')} </p>
+                            <Button className="my-4" variant="danger" onClick={() => this.removeBookFromList(this.state.addedBooks[0])}>Remove Book</Button>
+                          </Col>
+                        }
+                      </Row>
+                    </div>
+                    :
+                    <div>
+                      <h2 className="text-center my-4 px-4 ">Added Books</h2>
+                      <Row className="text-center px-3">
+                        {this.state.addedBooks.map(addedBook =>
+
+                          <Col md={3} className="px-2 my-0 border" key={addedBook.id}>
+                            <img className="img-responsive mt-3 p-0 rounded" src={addedBook.thumbnailLink} alt={addedBook.title} />
+                            <h5 className="mt-4"> {addedBook.title} </h5>
+                            <p className="my-1"> {addedBook.authors.join(', ')} </p>
+                            <Button className="my-5" variant="danger" onClick={() => this.removeBookFromList(addedBook)}>Remove Book</Button>
+                          </Col>
+                        )}
+                      </Row>
+                    </div>
+                )
+              }
+
+              <div className="text-center">
+                <Button className="text-center" variant="primary" onClick={async () => await this.handleSubmit()} >
+                  Add
+                </Button>
+              </div>
+            </Form>
+          </Modal.Body>
+        </Modal>
+      </div>
+    )
+  }
+}
+
+export default SearchBookModal;
```

Comment:
nit: You should avoid using default exports (https://google.github.io/styleguide/jsguide.html#named-vs-default-exports)

Reply:
Changed.